src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +154,23 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * Return last name for the specified gender.
+     *
+     * @access public
+     * @param string|null $gender A gender of the last name should be generated
+     *     for. If the argument is skipped a random gender will be used.
+     * @return string Last name
+     */
+    public function lastName($gender = null)
```

Comment:
Do I understand correctly or is the generated `lastName` always Male by default? Shouldn't it return a random gender?

Reply:
Good point, missed that one.

@aanfarhan 

Can you adjust, please?

Reply:
Alright, I'll add some adjustment to the code.